pysat/_orbits.py

```diff
@@ -270,7 +276,7 @@ def _equaBreaks(self, orbit_index_period=24.):
             ind = np.hstack((ind, ut_ind))
             ind = np.sort(ind)
             ind = np.unique(ind)
-            logger.info('Time Gap')
+            logger.info('Time Gap at locations ', ut_ind)
```

Comment:
```suggestion
            logger.info(' '.join(('Time Gap at locations', ut_ind)))
```

Reply:
Just realized this breaks things, since `ut_ind` is not a string.  Working on alternate versions, but the original may be best